### DISCUSSION TITLE
Cannot get text details for created hubs from my libraries

### DISCUSSION THREAD

**Discussion by DROIDshield** [jurialmunkey/skin.arctic.horizon.2#910] [Discussion]
Wonder if I might get some help. I've created accross the top of the screen a TV library hub and a second test movie hub and even though the movie and tv and movie hubs are setup exactly the same i get no information on them but do on the movie hub to the furthest right. All setting are the same and mirrored off the second movie hub apart from the paths for the content  see my pics and let me know if I'm  missing something. In another menu I need to activate to get these working....
Thanks.
![20230416_023800](https://user-images.githubusercontent.com/97023369/232261823-495878c6-7e3e-42bb-ae44-cdb36b5be493.jpg)
![20230416_023814](https://user-images.githubusercontent.com/97023369/232261825-167240a8-2460-421d-8794-cf1d80bb11c3.jpg)
![20230416_023823](https://user-images.githubusercontent.com/97023369/232261832-b5946b95-6e71-4068-84f3-f0b4a7c39a4e.jpg)

**Comment by jurialmunkey** [comment]
You need to set the "Fullscreen" dummy widget in the standard widgets group to force skinshortcuts to build if the fullscreen widget is your only widget. Otherwise skinshortcuts will not build the widget group.

**Reply by DROIDshield** [reply]
I have done this, mirrored from the movie hub that works... the fullscreen Widget is set. See the image from the test movie that doesn't populate and then the one from the Movie hub that does... they are exactly the same settings and so is the TV  but just different paths, this is why I can understand why they won't work with the same settings in all of the menus....

I don't really want to reset back to default because my AH2 works a dream... bit could it be a glitch that needs to be cleared?
![20230416_031352](https://user-images.githubusercontent.com/97023369/232262530-9ca0d175-54ca-442a-a828-2cdfc9fa24cb.jpg)
![20230416_031406](https://user-images.githubusercontent.com/97023369/232262536-603fe6dd-f8c2-49f8-adb6-066b5c993c00.jpg)
![20230416_031442](https://user-images.githubusercontent.com/97023369/232262542-73590fa9-9ca5-443c-9ca0-3610f2b545a0.jpg)

**Reply by DROIDshield** [reply]
Sorry forget the last message I sent.  I've found the widgets line. Filled it in and yes... working😄👍🏾
